src/basic/missing_syscall.h

```diff
@@ -33,39 +33,47 @@ static inline int missing_pivot_root(const char *new_root, const char *put_old)
 
 /* ======================================================================= */
 
-#if !HAVE_MEMFD_CREATE
+#if defined __x86_64__
+#  define fallback_NR_memfd_create 319
+#elif defined __arm__
+#  define fallback_NR_memfd_create 385
+#elif defined __aarch64__
+#  define fallback_NR_memfd_create 279
+#elif defined __s390__
+#  define fallback_NR_memfd_create 350
+#elif defined _MIPS_SIM
+#  if _MIPS_SIM == _MIPS_SIM_ABI32
+#    define fallback_NR_memfd_create 4354
+#  endif
+#  if _MIPS_SIM == _MIPS_SIM_NABI32
+#    define fallback_NR_memfd_create 6318
+#  endif
+#  if _MIPS_SIM == _MIPS_SIM_ABI64
+#    define fallback_NR_memfd_create 5314
+#  endif
+#elif defined __i386__
+#  define fallback_NR_memfd_create 356
+#elif defined __arc__
+#  define fallback_NR_memfd_create 279
+#else
+#  warning "memfd_create number unknown for your architecture"
```

Comment:
doesn't matter, but `memfd_create() system call numer unknown for your architecture` looks a bit better to me